base/_grid.scss

```diff
@@ -1,10 +1,16 @@
-// Grid
-// Example use: @include grid;
-@mixin grid {
-    display: -ms-grid;
+//Grid
+// Example use: @include grid(repeat(12, 1fr), auto, 32px, 75px);
+//Will produce a 12 column grid with auto row height and 32px column gap and 75px spacing between start and end
```

Comment:
Think these dev notes need updating since we split out the mixin from the implementation specifics